### DISCUSSION TITLE
Variant based on color scheme

### DISCUSSION THREAD

**Discussion by sharmapukar217** [heroui-inc/tailwind-variants#232] [General]
Say I have two variants of color say: primary and inverted and i want to apply primary Variant on light mode while inverted on dark mode. I want to know if there's something like responsive variants, also for colorScheme?

**Comment by draylegend** [comment]
```html
<style>
:root {
  color-scheme: light dark;
}

body {
  background: light-dark(#aef, #333)
}

.light {
  color-scheme: light
}

.dark {
  color-scheme: dark
}
</style>
```

Further reading: `oklch`, `from var(--color-primary)`
